public/app/features/dashboard/components/DashNav/IirisMaintenanceEditWizard.tsx

```diff
@@ -457,6 +462,9 @@ export class IirisMaintenanceEditWizard extends PureComponent<Props, State> {
           document.dispatchEvent(new Event('iiris-maintenance-update'));
         }, 2 * 60 * 1000);
 
+        // Enable the wizards buttons again
+        this.setState({ isLoading: false });
```

Comment:
Should the flag be cleared also on catch handler? So that the user can try again to save the maintenance if there has been eg. a network error that prevented saving?

Reply:
Yes it should. In commit af64ca964c473ba48d0ebaed0a2d535d5399b42c.